src/components/favourite-pollutant/favourite-pollutant.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { createStyles } from "@material-ui/core";
+import theme from "../../theme/theme";
+
+export const styles = createStyles({
+
+  button: {
+    color: theme.palette.primary.light,
+    background: theme.palette.primary.main,
+    zIndex: 1,
+    pointerEvents: "auto",
+  },
+
+  buttonholder: {
+    display: "flex",
+    width: "100%",
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+    justifyContent: "space-between"
+  }
+});
```

Comment:
```suggestion
  }
  
});
```
In stylesheets there should be empty line before the first style AND after the last style